JavaScript/JavaScript.sublime-syntax

```diff
@@ -359,28 +361,15 @@ contexts:
       set:
         - expect-semicolon
         - import-export-from
-        - export-list
-        - import-export-alias
-        - export-item
-
-  export-list:
-    - match: ','
-      scope: punctuation.separator.comma.js
-      push:
-        - import-export-alias
         - export-item
-    - include: else-pop
 
   export-item:
     - match: '\{'
       scope: punctuation.section.block.begin.js
       set: export-brace
-    - match: '{{non_reserved_identifier}}'
-      scope: variable.other.readwrite.js
-      pop: 1
     - match: '\*'
       scope: constant.other.js
```

Comment:
Does it make sense to apply recently introduced common wildcard scope to all those asterisks?

```suggestion
      scope: constant.other.wildcard.asterisk.js
```

Reply:
I'm sure it would — I just now read that RFC. Is it meant to apply to wildcard imports? That seems reasonable to me.

It would also be an easy change to apply in a standalone PR, or in an import-focused followup. I'm thinking in that PR I might re-consolidate some of the contexts that were split in the past.

Reply:
Wildcard scopes are meant for universal usage without any restrictions to certain kinds of expressions. Java also scopes wildcard imports this way.

I am fine with addressing that in a separate PR.